lib/atat-web-api-stack.ts

```diff
@@ -81,7 +88,16 @@ export class AtatWebApiStack extends cdk.Stack {
       const loadBalancer = new elbv2.ApplicationLoadBalancer(this, "LoadBalancer", {
         vpc: network.vpc,
         internetFacing: false,
+        deletionProtection: true,
```

Comment:
Nice add!

Reply:
Yeah, it'll be nice to make sure we don't change the DNS name unexpectedly. This is actually one of the `cdk-nag` rules.